### DISCUSSION TITLE
feat: Access Native Ui & webview UI  both at same time

### DISCUSSION THREAD

**Discussion by phoenix503** [ionic-team/capacitor#4793] [Q&A]
### Description
I want to add Side Navigation Drawer to my App as an Native UI not with the HTML code.
Please tell the possible ways to do that.

### Platform(s)
I have tried the same thing in Android , but unable to change the UI

### Preferred Solution
Their is a method name called `load()`

 ```
 private void load() {
        getApplication().setTheme(getResources().getIdentifier("AppTheme_NoActionBar", "style", getPackageName()));
        setTheme(getResources().getIdentifier("AppTheme_NoActionBar", "style", getPackageName()));
        setTheme(R.style.AppTheme_NoActionBar);
        setContentView(R.layout.bridge_layout_main);

        Logger.debug("Starting BridgeActivity");

        bridge = bridgeBuilder.addPlugins(initialPlugins).setConfig(config).create();

        this.keepRunning = bridge.shouldKeepRunning();
        this.onNewIntent(getIntent());

    }
```
It is a private method, if somehow i can change the setContentView in this code i can  able to add custom UI to Android App

### Alternatives
If not possible, please suggest me other Alternatives to use Native UI for Side Navigation Drawer & HTML code both at same time

### Additional Context
I am trying all these in Plain Capacitor App

**Comment by frederikbosch** [comment]
While I initially also tried the solution to override `load` or `onCreate` , because it is intuitively how it supposed to work. However, this became very hacky. I ended up with a far more easier solution: override `setContentView`.

```java
    public void setContentView(@LayoutRes int layoutResID) {
        if (layoutResID == com.getcapacitor.android.R.layout.bridge_layout_main) {
            super.setContentView(R.layout.activity_main);
        } else {
            super.setContentView(layoutResID);
        }
    }
```
